tester/device.go

```diff
@@ -0,0 +1,58 @@
+package tester
+
+import (
+	qt "github.com/frankban/quicktest"
+)
+
+const maxRegisters = 200
+
+// I2CDevice represents a mock I2C device on a mock I2C bus.
+type I2CDevice struct {
+	C    *qt.C
```

Comment:
I think I'd use an interface type rather than using *qt.C directly.

```
// Fataler is used by the I2CDevice type to abort when it's used in
// unexpected ways, such as reading an out-of-range register.
type Fataler interface {
    // Fatalf prints the Printf-formatted message and exits the current
    // goroutine.
    Fatalf(f string, a ...interface{})
}
```

Also, I don't think I'd bother exporting it - it's not a significant part of the user-visible API and the user should have access to it anyway.

Unfortunately I couldn't think of a better name for the interface :)

Reply:
I chose "Failer".